module/VuFindTheme/src/VuFindTheme/ThemeInfo.php

```diff
@@ -210,35 +210,80 @@ public function getThemeInfo()
         return $this->allThemeInfo;
     }
 
+    /**
+     * Determine if a variable is a string-keyed array
+     *
+     * @param mixed $op Variable to test
+     *
+     * @return boolean
+     */
+    protected function isStringKeyedArray($op) {
+        if (!is_array($op)) {
+            return false;
+        }
+
+        reset($op);
+        if (is_string(key($op))) {
```

Comment:
Further optimization: just `return is_string(key($op));` instead of an if and two different returns.